doc/tech/approach.md

```diff
@@ -0,0 +1,99 @@
+1. Provide a version controlled catalog of APIs, and their respective spans of control, capabilities and purpose, to facilitate predictable integration w/ a wide selection of fabric implementations
+1. Provide ability for any number of Operator-specific fabrics to power CNTT NFVI
+1. Enable RC's ability to realize mandated OVP qualification deliverables
+1. Unambiguously document the responsibilities of each CNTT constituency
+1. In cases where a VNF/CNF require HW layer resources it should be under the control of the Virtualization Layer
+1. It is important that the HW Infrastructure Manager, each VIM and each VNF/CNF could be managed by separate organizations
+1. Drive the industry towards convergence on ABIs supporting Cloud-Native implementations for SmartNICs
+
+> _**Editor's Note:** Consider moving objectives to a table_
+
+<a name="5.1.3"></a>
+### 5.1.3 Networking Requirements
+
+1. NFVI layer responsibilities will include:
+   * HW Infrastructure Manager shall provide an abstracted model of the allocated HW resources into each specific Virtualization domain
+     * Additionally, it is responsible for maintaining logical isolation between different instances of virtualization domain
+     * Some of the HW resources including networking resources shall be possible to be withheld from Virtualization domains to allow for scaling, spare parts and HW Composition within the HW Infrastructure Layer itself
+   * Virtualization layer shall provide the Cloud Tenants with an abstracted networking environment
+     * It is therefore responsible for maintaining isolation between Cloud Tenants
+
+An example of the layering described above is depicted in **Figure 1-3**, where the Virtualization layer manages the Overlay Networking (e.g., through VLAN allocation) and the HW Infrastructure manages the Underlay networking (e.g., through VxLAN VNI range allocations).
+<p align="center"><img src="./figures/RMCH-01_Network_Layering_v0-3.png" alt="Network Layering" title="Network Layering" width="100%"/></p>
+<p align="center"><b>Figure 1-3:</b> Network Layering</p>
+
+<a name="5.1.4"></a>
+### 5.1.4 Initial Logistical Steps
+
+> _**Editor's Note:** The purpose in this section is to communicate the responsibilities of the CNTT levels, as they pertain to the networking solution, and how they relate (i.e. what goes in each bucket). The initial content below needs to be expanded with examples of material representative of the scope for each bucket, and refined. It is expected this area be enhanced to help authors in various CNTT levels understand their purview, and ultimately be deleted._
+
+As with most CNTT subsystems, responsibility for Objectives, Requirements, Guidelines, etc. is divided among the CNTT constituencies, as follows:
+- **Tech:** High-level strategy and other informative writings
+  - Ex. Executive summary, considerations, intent, vision, etc.
```

Comment:
Can I suggest using e.g. instead of Ex. (not clear whether that means example or excluded).

Reply:
I agree with @tomkivlin.  I know this section was moved wholesale from the RM (and 'ex' was in the document text there) but I think it's a useful and minor clarification similar to some of the other ones that have been made in this transition.

Reply:
agree, fixed in here: https://github.com/cntt-n/CNTT/pull/1267/commits/14c7b624cfcc3338fa4cf9363fd418b1f6fb17cb